docs(kfp): add local option to KFP installation overview (#2376)
kubeflow/website: content/en/docs/pipelines/installation/overview.md

1. Do you want to use other Kubeflow components in addition to Pipelines?

    If yes, choose the [full Kubeflow deployment](#full-kubeflow-deployment).
1. Can you use a cloud/on-prem Kubernetes cluster?

>     If no, you can try Kubeflow Pipelines on a local Kubernetes cluster following [Deploying Kubeflow Pipelines on a local cluster](/docs/pipelines/installation/localcluster-deployment) for learning and testing purposes.

Reviewer: ```suggestion
    If you can't, you should try using Kubeflow Pipelines on a local Kubernetes cluster for learning and testing purposes by following the steps in [Deploying Kubeflow Pipelines on a local cluster](/docs/pipelines/installation/localcluster-deployment).
```

WDYT? cc @alfsuse @RFMVasconcelos 

Thanks @Bobgy

Author: LGTM! Let's wait for their opinions too

Reviewer 2: /lgtm sorry I was not much very active these days.
